cmd/influxd/launcher/launcher.go

```diff
@@ -339,6 +346,8 @@ type Launcher struct {
 	enableNewMetaStore   bool
 	newMetaStoreReadOnly bool
 
+	featureFlags string
```

Comment:
Hey Gavin, does the list get stringified at some point or is it always a string?

Reply:
It's always a string and I split the list on "," myself, but it just occurred to me viper offers support for `[]string` and will do the splitting for you, so im going to refactor this to use that, which will make this bit simpler https://github.com/influxdata/influxdb/pull/17851/files#diff-5fb5403debd4552b20d5d324f7626779R38